pkgs/data/themes/andromeda-gtk-theme/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+stdenvNoCC.mkDerivation {
+  pname = "andromeda-gtk-theme";
+  version = "250751a";
+
+  src = fetchFromGitHub {
+    owner = "EliverLara";
+    repo = "Andromeda-gtk";
+    rev = "250751a546dd0fa2e67eef86d957fbf993b61dfe";
+    sha256 = "0k86c4jm7mw9b2rg7rmg0rbbklh2nw63v50j0rfgzn6nz27zs6kv";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/themes/Andromeda-gtk
+    cp -r $src/* $out/share/themes/Andromeda-gtk
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = with lib; {
    description = "An elegant dark theme for gnome, mate, budgie, cinnamon, xfce";
```
See https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes